### PR TITLE
Added ITP screen

### PR DESCRIPTION
**Pull request by alexalghisi**

### CODE REVIEW

File: src/components/documentsList.js [hunk 1]
```diff
@@ -33,14 +35,26 @@ const DocumentsList = props => {
...
     }
 
     return (
       <View style={styles.item}>
-        <Image style={styles.item} source={{ uri: item.imageURI }} />
+        <TouchableOpacity
+          style={styles.item}
+          onPress={() => {
+            props.navigation.navigate("DocumentAuto", {
+              props,
```
Comment (Sfirleaalex) [inline src/components/documentsList.js]: This is wrong on many levels. Send only what you need

File: src/screens/DocumentAuto.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+import Colors from "../constants/colors";
+import Dimensions from "../constants/dimensions";
+
+const DocumentAuto  = props =>
+{
+  const type = props.navigation.getParam("type");
+  const expire = props.navigation.getParam("expire");
+  const imageURI = props.navigation.getParam("imageURI");
+  return <View style={styles.container}>
+    <Text style={{color:'blue'}}>
```
Comment (Sfirleaalex) [inline src/screens/DocumentAuto.js]: Move it in styles and create a color for blue

File: src/components/documentsList.js (hunk 1 above)
Comment (alexalghisi) [reply src/components/documentsList.js]: 😂 ok

File: src/screens/DocumentAuto.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+           {type}
+       </Text>
+      <Text style={[styles.itemText, styles.dateTextStyle]}>
+        {expire}
+      </Text>
+   </View>
+};
+
+const styles = StyleSheet.create({
+  item: {
```
Comment (Sfirleaalex) [inline src/screens/DocumentAuto.js]: item is not a good name for a style which corresponds to image. Also create constants for width and height

File: src/screens/DocumentAuto.js (hunk 2 above)
Comment (Sfirleaalex) [reply src/screens/DocumentAuto.js]: Why you have an image inside text?

Comment (alexalghisi) [reply src/screens/DocumentAuto.js]: Good observation 👍

File: src/components/documentsList.js [hunk 4]
```diff
@@ -33,14 +35,25 @@ const DocumentsList = props => {
...
-  const renderItem =  ({ item })  => {
+  const renderItem = ({ item }) => {
     if (item.empty === true) {
       return <View style={[styles.item, styles.itemInvisible]} />;
     }
 
     return (
       <View style={styles.item}>
-        <Image style={styles.item} source={{ uri: item.imageURI }} />
+        <TouchableOpacity
```
Comment (Sfirleaalex) [inline src/components/documentsList.js]: I think all the element should be clickable

File: src/components/documentsList.js [hunk 5]
```diff
@@ -33,14 +35,25 @@ const DocumentsList = props => {
...
+          style={styles.item}
+          onPress={() => {
```
Comment (Sfirleaalex) [inline src/components/documentsList.js]: Extract into a function

File: src/components/documentsList.js (hunk 4 above)
Comment (alexalghisi) [reply src/components/documentsList.js]: The Image ?

**Pull request merged**